master/reference/felix/configuration.md

```diff
@@ -21,33 +21,33 @@ The full list of parameters which can be set is as follows.
 
 | Configuration parameter           | Environment variable                    | Description  | Schema |
```

Comment:
Would be nice to reword bullets 3 and 4 [here](https://github.com/projectcalico/calico/pull/1587/files#diff-51781dae74db83cc7d6d3f2a04fe4f0aR10) so they don't talk about etcd, since this isn't etcd specific.

e.g.

```
3. Host specific FelixConfiguration resources
4. The Global 'default' FelixConfiguration resource
```